test/SIL/Serialization/keypath.sil

```diff
@@ -49,6 +49,13 @@ public struct External<T> {
   init()
 }
 
+struct T {
+  var x: (Int, Int)
+  var y: (a: Int, b: Int, c: Int)
+  let z: (Int, Int)
+  let w: (Int, C)
+}
```

Comment:
Might be nice to add a test case where `T` is generic and the tuples are dependent on the generic argument, to ensure the generic substitutions propagate properly through the tuple element.